.ci/windows-steps.yaml

```diff
@@ -14,6 +14,7 @@ steps:
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install unofficial-flayan-cereal -o $(Agent.ToolsDirectory)
+    vcpkg install ensmallen:x64-windows
```

Comment:
I'm not sure if this installed to the right place---at least, CMake wasn't able to find it.  I wonder if you can use nuget to install instead?  I'm not sure, I don't know much about Windows configuration... I assume also STB will be necessary here too.

Reply:
The same as you, I have no clue on what is happening on the Windows side, I will use `nuget` and see what happens, for stb, I think we need to download it too.